standalone/src/chain_spec.rs

```diff
@@ -299,6 +299,10 @@ fn testnet_genesis(
                 (CurrencyId::DOT, FixedU128::checked_from_rational(110, 100).unwrap()),
                 (CurrencyId::KSM, FixedU128::checked_from_rational(110, 100).unwrap()),
             ], /* 110% */
+            system_collateral_ceiling: vec![
+                (CurrencyId::DOT, 1_000_000_000_000_000),
+                (CurrencyId::KSM, 1_000_000_000_000_000),
```

Comment:
These should also be derived automatically from the expected number of decimal places. It's worth noting that DOT has 10 whereas KSM has 12.